.github/workflows/release_chart.yml

```diff
@@ -19,6 +19,6 @@ jobs:
         with:
           token: ${{ secrets.ACTION_PAT }}
           charts_dir: .
```

Comment:
charts_dir: helm is required

Reply:
@syedsalman3753, charts_dir need not to be changed as it was tested in local and it working fine with existing charts_dir mentioned.